pkg/operators/eventgen/eventgen.go

```diff
@@ -0,0 +1,163 @@
+package eventgen
+
+import (
+	"fmt"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/params"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/logger"
+)
+
+const (
+	name                = "eventgen"
+	ParamEventGenEnable = "eventgen-enable"
+	ParamEventGenType   = "eventgen-type"
+	ParamEventGenTarget = "eventgen-target"
+)
+
+type Generator interface {
+	Generate(target string) (string, string, string, error)
+	Cleanup(podName string) error
+}
+
+type eventGenOperator struct{}
+
+func (e eventGenOperator) Name() string {
+	return name
+}
+
+func (e eventGenOperator) Init(params *params.Params) error {
+	return nil
+}
+
+func (e eventGenOperator) GlobalParams() api.Params {
+	return nil
+}
+
+func (e eventGenOperator) InstanceParams() api.Params {
+	return api.Params{
+		&api.Param{
+			Key:         ParamEventGenEnable,
+			Description: "Enable event generation",
+			TypeHint:    "bool",
+		},
```

Comment:
Sorry for this, on my side by putting it in global parameter like this 
```
func (e eventGenOperator) GlobalParams() api.Params {
	return api.Params{
		&api.Param{
			Key:         ParamEventGenEnable,
			Description: "Enable event generation",
			TypeHint:    "bool",
			// DefaultValue: "true",
		},
	}
}

```
Its showing `eventgen is not enabled`. So I am not able to test it locally. It will be in my TODO and after finishing other changes, i will change this to global param.

Reply:
How are you testing it? With `ig` or `ig` + `gadgetctl`?

Reply:
only ig, by creating `ig` binary and run it.